### DISCUSSION TITLE
Using fancybox alert 4.0.8 JS

### DISCUSSION THREAD

**Discussion by gustavopalinka** [fancyapps/ui#197] [General]
Good morning, recently I'm upgrading the **Fancybox 1.3 to 4.0.8** but I found some issues that I need help.

In Fancybox 1.3 one of most used resources that I use is the messages in pop-up through of a function:
 
Like this:
```
function AlertFancyBox($msg) {
      if ($msg == "error_zipcode") {
            $message = '<div class="alert-4"><div class="alert-info"><p><span class="text-alert"><strong>Error: ZIP code not found</strong>< br>Please check the zip code provided.</span></p></div>'+ $button +'</div>';
      } else if ($msg == "change_password") {
              $message02 = '<div class="alert-4"><div class="alert-info"><p><span class="text-alert">Please enter your current password for change processing. </span></p></div>'+ $button +'</div>';
     }
... many other messages here....
```

And a just call the message in JS.
`AlertFancyBox("change_password"); `

This messages don't work in FancyBox 4.0.8, my doubt is how I can keep using this alerts that I created in new version of FancyBox 4.0.8?

**Comment by fancyapps** [comment]
Hi,

Did you check documentation? The first example here - https://fancyapps.com/docs/ui/fancybox/api - shows how to display HTML content.
